pyspecProcScripts/third_level/analyze_square_refl.py

```diff
@@ -1,32 +1,7 @@
-""" Analyzes square wave reflection data
-========================================
+""" Analyze square wave reflection data
+=======================================
 Analyzes data acquired by applying a square wave pulse to an empty 
 or filled NMR probe and acquiring the reflection profile.
-
-Parameters
-==========
-d:      nddata
-label:  str
-        appropriate name for dataset.
-frq_bw: int
-        bandwidth in Hz.
-keep_after_pulse:   int
-                    Amount of the time axis after the pulse
-                    that you want to remain-changes depending 
-                    on the Q.
-blip_range:         lst
-                    Range in which the signal blip exists.
-show_analytic_signal_phase: bool
-                            Option on final plot to show the 
-                            phase of the signal.
-show_analytic_signal_real:  bool
-                            Option on final plot to show the 
-                            real of the signal for comparison.
-
-Returns
-=======
-Plots of the reflected square wave with a fit that calculates
-the Q for the probe being tested
```

Comment:
I had deleted this b/c the function doesn't return anything (return is very specifically supposed to tell you about the objects that are returned)

**BUT** I realized this is actually a good short description for your function, so you probably want to put this at the beginning of your function docstring (which is missing a description)